client/src/services/resourceService.js

```diff
@@ -16,6 +16,13 @@ export default class ResourceService {
 		return [];
 	}
 
+	async getOne(id) {
+		const res = await this.fetch(`${ResourceService.ENDPOINT}/${id}`);
```

Comment:
If fetch fails, an unhandled exception will be thrown. Wrapping the request in a try-catch block allows for error handling.